Contact.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles/main.css">
+  <title>Contact me</title>
+</head>
+
+<body>
+
+
+  <header class="site-header">
+    <span class="logo">ALKHADRA</span>
```

Comment:
Since `span` is an inline element, it will be best to maybe wrap the logo in some kind of bloxk level element as well for easier layout.